app/decorators/answer_finder.rb

```diff
@@ -24,8 +24,7 @@ def answers_for_question(question)
       .joins(:form_item)
       .where(response_id: responses.map(&:id))
       .where("form_items.question_id" => question.id)
-      .where("inst_num = 1")
-      .where("answers.rank = 1")
+      .where("answers.old_rank = 1")
```

Comment:
Actually I think instead of this we can just do `order(rank: :desc)` and then if there are multiple matches for the question, the one with the lowest rank will end up being the last passed to `index_by` so it will be the result.